src/providers/wms/qgswmscapabilities.cpp

```diff
@@ -783,6 +783,34 @@ void QgsWmsCapabilities::parseLegendUrl( QDomElement const &e, QgsWmsLegendUrlPr
   QgsDebugMsg( QStringLiteral( "exiting." ) );
 }
 
+void QgsWmsCapabilities::parseMetadataUrl( QDomElement const &e, QgsWmsMetadataUrlProperty &metadataUrlProperty )
+{
+
+  QDomNode n1 = e.firstChild();
+  while ( !n1.isNull() )
+  {
+    QDomElement e1 = n1.toElement(); // try to convert the node to an element.
```

Comment:
```suggestion
    QDomElement e1 = n1.toElement();
```
(it's redundant -- the code already describes exactly this :wink:  )

Reply:
Sure